central/sensor/service/connection/sensorevents.go

```diff
@@ -65,6 +72,10 @@ func (s *sensorEventHandler) addMultiplexed(ctx context.Context, msg *central.Ms
 	var workerType string
 	switch evt := msg.Msg.(type) {
 	case *central.MsgFromSensor_Event:
+		if msg.GetEvent() == nil {
```

Comment:
Would we even get this far if this were true? Maybe we can get rid of the outer switch and just check this at the beginning of the function?

It could be something like
```
evt := msg.GetEvent()
if evt == nil { ... }

switch evt.Resource.(type) { ... }
```